spec/data-model/README.md

```diff
@@ -148,20 +148,27 @@ interface Option {
 }
 ```
 
-A `Reserved` represents an _expression_ with a _reserved_ _annotation_.
-The `sigil` corresponds to the starting sigil of the _reserved_.
+An `Unsupported` represents an _expression_ with a _reserved_ or _private-use_ _annotation_.
```

Comment:
Intentional, because we're at least so far not referring to data model constructs from elsewhere in the spec, and their names would very closely conflict with syntax rules. The data model is currently entirely built on top of the rest, and we should be careful about establishing a dependency on it from elsewhere.